src/domain/participant/index.js

```diff
@@ -94,63 +101,76 @@ const getByName = async (name) => {
 }
 
 const participantExists = (participant, checkIsActive = false) => {
+  logger.debug('checking if participant exists', { participant, checkIsActive })
   if (participant) {
     if (!checkIsActive || participant.isActive) {
       return participant
     }
+    logger.error('participant is inactive', { participant })
```

Comment:
maybe, `warn` severity is more appropriate here?

Reply:
done